code/modules/mob/living/carbon/human/species_types/lizardpeople.dm

```diff
@@ -18,6 +18,7 @@
 	exotic_bloodtype = "L"
 	disliked_food = GRAIN | DAIRY
 	liked_food = GROSS | MEAT
+	ass_pic = "unathi"

```

Comment:
Use standard male/female pics? There doesn't seem to be a gender distinction for unathi and the butt image looks very ugly.